### DISCUSSION TITLE
Using Signatures

### DISCUSSION THREAD

**Discussion by Gordonbp** [elementary/mail#792] [Q&A]
Just started using Elementary mail - there doesn't seem to be a signature option. Am I missing something?

**Comment by marbetschar** [comment]
That's correct: For the time being there is no support for signatures in Mail. See https://github.com/elementary/mail/issues/616 for the corresponding issue.

There is quite a few stuff on my plate related to improvements of Mail, but unfortunately I did not found the time yet due to my studies. I hope to get to work on some stuff during this summer break, no promises though if I get around working on signatures :)

Meanwhile: Since Mail and Evolution share the same backend, you might want to experiment with the signature options in Evolution - maybe you have some luck getting things to work in Mail with this approach as well (not sure though).

When experimenting, please make sure you install Evolution as `*.deb` (using apt-get) - this way it allows you to tinker with the account settings which are used by Mail as well.
